IPython/core/inputtransformer2.py

```diff
@@ -0,0 +1,564 @@
+"""Input transformer machinery to support IPython special syntax.
+
+This includes the machinery to recognise and transform ``%magic`` commands,
+``!system`` commands, ``help?`` querying, prompt stripping, and so forth.
```

Comment:
Perhaps add to comment:

Added: IPython 0.7. Replaces inputsplitter and inputtransformer which were deprecated in 0.7.

Reply:
Done.